cdap-gateway/src/test/java/co/cask/cdap/gateway/router/NettyRouterPipelineTest.java

```diff
@@ -17,33 +17,11 @@
 package co.cask.cdap.gateway.router;
 
 import co.cask.cdap.AllProgramsApp;
-import co.cask.cdap.api.common.Bytes;
```

Comment:
Changes in this class are mostly:
- moving the RouterResource and ServerResource out as separate classes, to be reused by other tests
- renaming some variable to conform to our coding standards